docs/src/pages/components/authenticator/index.page.mdx

```diff
@@ -236,6 +236,8 @@ For your [configured social providers](https://docs.amplify.aws/lib/auth/social/
   <Authenticator socialProviders={['amazon', 'apple', 'facebook', 'google']} />
 </Example>
 
+_[Step by step video](https://youtu.be/8KwZNn56F78) on setting up social providers._
```

Comment:
Should we embed the video rather than linking off to it?

Reply:
I just want it linked right now instead of embedding it. That way it doesn't distract from the rest of the help articles.